common/src/Microsoft.Azure.IIoT.Hub.Module.Framework/src/Hosting/ModuleHost.cs

```diff
@@ -538,10 +540,11 @@ public sealed class ModuleHost : IModuleHost, ITwinProperties, IEventEmitter,
         private readonly SemaphoreSlim _lock = new SemaphoreSlim(1, 1);
         private readonly Dictionary<string, VariantValue> _reported =
             new Dictionary<string, VariantValue>();
+        private readonly string ModuleGuid = Guid.NewGuid().ToString();
```

Comment:
>ModuleGuid [](start = 32, length = 10)

coding conventions: either _moduleGuid, or make it a property.

Or maybe better have it globally  visible as part of the Instance so that all the metrics can reuse it. Then we would know all thing were generated by a single runtime instance